core/node/groups.go

```diff
@@ -122,6 +122,12 @@ func LibP2P(bcfg *BuildCfg, cfg *config.Config) fx.Option {
 			logger.Error("Use the `Swarm.Transports.Network.Relay' config field instead")
 		}
 	}
+	//nolint
+	if cfg.Swarm.EnableRelayHop {
+		logger.Error("The `Swarm.EnableRelayHop` config field is deprecated.\n" +
```

Comment:
It is noop, so better to be explicit it does nothing :)
```suggestion
		logger.Error("The `Swarm.EnableRelayHop` config field is ignored.\n" +
```